packages/ncids-js/src/components/usa-footer/utils/footer-back-to-top.component.ts

```diff
@@ -0,0 +1,69 @@
+/**
+ * NCI Back To Top
+ */
+export class NCIBackToTop {
+	/** Default Element */
+	protected element: HTMLElement;
+	/** Callback for handling the toggle functionality.  */
+	private scrollListener: EventListener = () => this.handleScroll();
+
+	/** Map object of the component. */
+	private static _components: Map<HTMLElement, NCIBackToTop> = new Map<
+		HTMLElement,
+		NCIBackToTop
+	>();
+
+	protected constructor(element: HTMLElement) {
+		this.element = element;
+
+		const existingComponent = NCIBackToTop._components.get(this.element);
+		if (existingComponent) {
+			existingComponent.unregister();
+		}
+		NCIBackToTop._components.set(this.element, this);
+
+		this.initialize();
+	}
+
+	public static create(element: HTMLElement): NCIBackToTop {
```

Comment:
Shouldn't be a component (more like footer collapse than NCI subscribe that the footer uses)